### DISCUSSION TITLE
¿How to increase the RAM and processors of netbox?

### DISCUSSION THREAD

**Discussion by sr-emm** [netbox-community/netbox-docker#639] [Getting Help] [answered]
Hi! total newbie with docker here, I'm trying to find out how to upgrade my netbox with better resources to work but can't find it directly in documentation or this forum.

Can anyone point me in the right direction?

Regards!

**Comment by ryanmerolle** [comment, marked as answer]
Assuming you have not setup resource limits in docker/docker-compose , then it will just leverage all available resources on the underlying host.

https://docs.docker.com/compose/compose-file/compose-file-v3/#resources
